reference/template-reference.md

```diff
@@ -0,0 +1,72 @@
+---
+title: template for reference <!-- Describe the title of your article by replacing "template for reference" with the page name you want to publish. -->
+author: The Good Docs Project
+uri-repo: https://github.com/projectname
+company: REPLACEME
+product: <!-- Set the product parameter to add the product name here. -->
+app: <!-- Insert the name of the app here. -->
+tags: <!-- Set other global keywords here like app name and product name or any other likely labels. These are comma-separated tags.-->
+---
+
+# Reference template
+
+## Overview
+Summarise what this reference article is about. What you put here is reused in the Overview section and include in HTML description tags.
+
+<!--
+Begin your Reference topic in this section. 
+For help with writing and structuring a reference article, see the README.md in the template directory for basic guidelines and links.
+Check out https://www.markdownguide.org/basic-syntax/ if you get stuck with AsciiDoc syntax.
+-->
+
+## Parameters
+Table 1. Example parameter reference template
+
+|Name |Type |Required |Description |
+|:--- |:--- |:--- |:--- |
+|productCode|`string`|Yes|Code of the document product to return the schema for. <br> <ul><li>Here is a bulleted list with a \| (pipe) inside a table.</li><li>Another bulleted list.<ul><li>An indented list</li></ul></li><li>Back to the list.</li></ul> |
+|||||
+
+## Field reference
+
+![Field reference](https://docs.oracle.com/cloud/latest/marketingcs_gs/OMCAA/Resources/Images/Forms/kbf2.x1.jpg =250x350)
```

Comment:
You asked on our channel about why the image wasn't resizing.  This is because the url isn't formatted properly with `=250x350` dangling.  And isn't clear if that url is a dynamic image url capable of resizing.  OR, if that is just a static-url.  Sorry this comment may seem confusing, happy to help clarify here if you need more background on what is wrong.

Reply:
Thanks for explaining why. 😀 Now I understand a little bit more. I will remove `=250x350` and add a comment about resizing for now.